src/runtime_src/core/pcie/windows/device_windows.cpp

```diff
@@ -617,18 +577,18 @@ struct info
   static result_type
   mgmt(const xrt_core::device* device, key_type key)
   {
-    auto init_device_info = [](const xrt_core::device* dev) {
-      XCLMGMT_IOC_DEVICE_INFO info = { 0 };
-      mgmtpf::get_device_info(dev->get_mgmt_handle(), &info);
+    auto init_pcie_info = [](const xrt_core::device* dev) {
```

Comment:
This is a major change.

What was this needed?

Reply:
Pcie info was originally part of "XCLMGMT_IOC_DEVICE_INFO" struct which is not available in golden image. So we decided to decouple them so that pcie info is available for golden image.

Reply:
Thank you for answering my questions here.